apps/hobbies-helsinki/config/jest/TestProviders.tsx

```diff
@@ -47,9 +50,11 @@ function TestProviders({ mocks, children, router }: Props) {
       <MockedProvider mocks={mocks} addTypename={false}>
         <EventsConfigProviderWithMockedApolloClient router={router}>
           <RHHCConfigProviderWithMockedApolloClient router={router}>
-            <RouterContext.Provider value={{ ...router, ...mockRouter }}>
-              {children}
-            </RouterContext.Provider>
+            <NavigationContext.Provider value={{}}>
```

Comment:
Yes, I've tried to move the TestProvider couple of times already, but there are quite many imports from the application, which would not be available in the common-test-package.

```
import EventsConfigProvider from '../../src/common-events/configProvider/ConfigProvider';
import eventsDefaultConfig from '../../src/common-events/configProvider/defaultConfig';
import { ROUTES } from '../../src/constants';
import { getLocalizedCmsItemUrl } from '../../src/utils/routerUtils';
import { initI18n as i18n } from './initI18n';

const CMS_API_DOMAIN = 'tapahtumat.cms.test.domain.com';
```

Therefore, a lot of refactoring would be needed and not much would be included in the common part. Could be done at some point, but I don't see much advantage of it since most of the content comes from the application specific configuration.

Reply:
Maybe this could be done at some point later